awsclient/iam.go

```diff
@@ -38,7 +38,7 @@ type rolePrincipal struct {
 	Service string
 }
 
-func (c *client) MakeLambdaFunctionDefaultPolicy() (string, error) {
+func (c *client) EnsureLambdaRoleAndPolicy(log Logger) (string, error) {
```

Comment:
this really ought to be in `awsapps` but some other time, maybe once I figure out the API Gateway provisioning